resources/views/partials/main-nav.blade.php

```diff
@@ -15,6 +15,6 @@
         <h6 class="dropdown-header">Community packages</h6>
         <a class="dropdown-item" target="_blank" href="https://github.com/dave-redfern/laravel-doctrine-behaviours">Laravel Doctrine Behaviours</a>
         <a class="dropdown-item" target="_blank" href="https://github.com/bharley/laraveldoctrine-hashids">Hashids</a>
-        <a class="dropdown-item" target="_blank" href="https://github.com/maxbrokman/SafeQueue">SafeQueue</a>
+        <a class="dropdown-item" target="_blank" href="https://github.com/digband/safe-queue">SafeQueue</a>
```

Comment:
I am getting 404 for this page. Is it broken?
`https://github.com/digband/safe-queue`